README.md

```diff
@@ -0,0 +1,60 @@
+# iOS enhanced media player
+
+Repo for application in iOS for media player
+
+## Table of Contents
+
+- [Motivation](#motivation)
+- [Minimum deployment target](#minimum-deployment-target)
+- [Integration](#integration)
+- [Features](#features)
+- [Visual references](#visual-references)
+
+## Motivation
+
+A template project to build mobile iOS media player in Swift. 
+
+## Minimum deployment target
+
+iOS 15.0 
+
+## Integration
+
+How to integrate with an existing project (SPM or CocoaPod)
+
+- Create a new or open a project on Xcode, then:
+
+SPM:
+- Add a new package via github, using the branch `main`
+
+CocoaPods:
+- On your project create the `Podfile` bellow
+- Run `pod install`
+
+```
+platform :ios, '15.0'
+
+target 'SampleApp' do
+    pod 'EnhancedMediaPlayer', :git => 'https://github.com/profusion/ios-enhanced-media-player', :branch => 'main'
+end
+```
+
```

Comment:
In this section add a sample code on how to instantiate the player from the package in a SwiftUI `View`